packages/drivers/odsp-driver/src/test/epochTests.spec.ts

```diff
@@ -48,6 +48,14 @@ describe("Tests for Epoch Tracker", () => {
         await epochTracker.removeEntries().catch(() => {});
     });
 
+    it("defaultCacheExpiryTimeoutMs <= maximumCacheDurationMs policy", () => {
+        // This is the maximum allowed value per the policy - 5 days
+        const maximumCacheDurationMs: Exclude<IDocumentStorageServicePolicies["maximumCacheDurationMs"], undefined> =
+            432000000;
```

Comment:
Nit: I prefer `5 * 24 * 60 * 60 * 1000` as it's easier to spot if someone fat fingered a number

Reply:
Compiler won't let me.  As long as I typed it correctly in the type definition, the compiler also won't let me typo it elsewhere :)